infra/base-images/base-runner/run_fuzzer

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash -eu
-# Copyright 2016 Google Inc.
+# Copyright 2018 Google Inc.
```

Comment:
nit: I guess you intended to mean the year in compile_perffuzz file

Reply:
+1

Reply:
Yes Sir!, you are right!